ci/gitlab/.gitlab-ci.yml

```diff
@@ -127,7 +127,7 @@ debian-build+static:
     GIT_CLONE_PATH: $CI_BUILDS_DIR/aktualizr-debian-$CI_JOB_ID
     GIT_SUBMODULE_STRATEGY: 'recursive'
 
-    TEST_BUILD_DIR: 'build-debian-testing'
+    TEST_BUILD_DIR: 'build-ubuntu-focal'
```

Comment:
The job is still called `debian-build+static`

Reply:
Good call, fixed.